src/test/java/uk/gov/companieshouse/api/accounts/service/impl/CompanyServiceImplTests.java

```diff
@@ -68,16 +72,17 @@ void getCompanyProfileThrowsApiErrorResponseException() throws Exception {
         when(mockCompanyGet.execute()).thenThrow(ApiErrorResponseException.class);
 
         assertThrows(ServiceException.class, () ->
-            companyService.getCompanyProfile(COMPANY_NUMBER));
+                companyService.getCompanyProfile(COMPANY_NUMBER));
     }
 
     @Test
     @DisplayName("Get Company Profile - Throws URIValidationException")
-    void getBalanceSheetThrowsURIValidationException() throws ApiErrorResponseException, URIValidationException {
+    void getBalanceSheetThrowsURIValidationException() throws ApiErrorResponseException,
```

Comment:
Method name should be specific to company profile (I know this was pre-existing, reformatting just highlighted the problem)